synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -207,6 +213,13 @@ MainWindow::init_menus()
 		sigc::ptr_fun(studio::App::dialog_help)
 	);
 
+#if GTKMM_MINOR_VERSION >= 20
```

Comment:
Wouldn't GTK_CHECK_VERSION be better?
Anyway, how can you use GTKMM_MINOR_VERSION? I did a `grep -R GTKMM_MI /usr/include/gtkmm-3.0/` and couldn't find this macro.

Reply:
My IDE pop that autocomplete feature up when I started typing GTKMM. Will check it later :)

Reply:
Changes on the way :)